symsorter/src/app.rs

```diff
@@ -335,9 +332,9 @@ pub fn main() -> ! {
         Ok(()) => std::process::exit(0),
         Err(error) => {
             eprintln!("{}: {}", style("error").red().bold(), error);
-            for cause in error.iter_causes() {
+            error.chain().skip(1).for_each(|cause| {
```

Comment:
Let's stick to the regular `for` syntax:

```suggestion
            for cause in error.chain().skip(1) {
```